src/main/java/software/amazon/smithy/lsp/SmithyTextDocumentService.java

```diff
@@ -254,8 +262,30 @@ private String findToken(String path, Position p) throws IOException {
     public CompletableFuture<Either<List<? extends Location>, List<? extends LocationLink>>> definition(
             DefinitionParams params) {
         try {
+            List<Location> locations;
```

Comment:
It would be nice to have a description of what this is doing behaviorally.

Reply:
Added with an example.